enterprise/frontend/src/embedding-sdk/types/ui.ts

```diff
@@ -0,0 +1,9 @@
+import type { Icons } from "metabase/ui";
+
+/**
+ * Needed to properly unwrap all values of the `IconName` type in documentation
+ * @inline
```

Comment:
Ah, does `@inline` treat this as an inline comment and exclude this from the documentation?